go.mod

```diff
@@ -8,11 +8,11 @@ require (
 	github.com/kylelemons/godebug v1.1.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.5.1
-	github.com/verrazzano/verrazzano-coh-cluster-operator v0.0.0-20200921120302-d4040fa2f3c4
-	github.com/verrazzano/verrazzano-crd-generator v0.0.0-20201006151822-9842cefdda24
-	github.com/verrazzano/verrazzano-helidon-app-operator v0.0.0-20200921121606-448c29f0ce58
-	github.com/verrazzano/verrazzano-monitoring-operator v0.0.0-20200925113618-cc189235ad3d
-	github.com/verrazzano/verrazzano-wko-operator v0.0.0-20201005195304-9238fa0e1f4a
+	github.com/verrazzano/verrazzano-coh-cluster-operator v0.0.10
```

Comment:
These dependencies should not be changing.  The psuedo tag is what is being used now.

Reply:
Yeah, I didn't understand that and and don't understand why it was updated? Do you know how to fix this?

Reply:
You should be able to replace with what is in master

Reply:
ok, I got this fixed and with checkin the new go.mod, go.sum

Reply:
ok, resolved in last commit